dlt/sources/helpers/rest_client/utils.py

```diff
@@ -0,0 +1,34 @@
+from typing import Tuple
+
+from dlt.common import logger
+from dlt.extract.source import DltSource
+
+
+def join_url(base_url: str, path: str) -> str:
+    if base_url is None:
+        raise ValueError("Base URL must be provided or set to an empty string.")
+
+    if base_url == "":
+        return path
+
+    if path == "":
+        return base_url
+
+    # Normalize the base URL
+    base_url = base_url.rstrip("/")
+    if not base_url.endswith("/"):
+        base_url += "/"
+
+    return base_url + path.lstrip("/")
+
+
+def check_connection(
```

Comment:
I'd remove those and keep in verified sources. WDYT?

Reply:
Yes, it's here by mistake

Reply:
Fixed in 9b3d353b9728c1bbf3608135ecf6f5cc6e70cf3a